tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -80,18 +98,30 @@ private String createStaticResourceResponse(String path) throws IOException {
         return create200Response(Files.readString(staticResource), ContentType.of(fileExtension));
     }
 
-    private void validateAccount(String account, String password) {
-        Optional<User> optionalUser = InMemoryUserRepository.findByAccount(account);
+    private String login(HttpRequest request) {
+        Optional<User> optionalUser = InMemoryUserRepository.findByAccount(request.getBody("account"));
         if (optionalUser.isEmpty()) {
             log.info("존재하지 않는 유저입니다.");
-            return;
+            return create302Response("/401.html");
         }
         User user = optionalUser.get();
-        if (!user.checkPassword(password)) {
+        if (!user.checkPassword(request.getBody("password"))) {
             log.info("비밀번호가 일치하지 않습니다.");
-            return;
+            return create302Response("/401.html");
         }
         log.info(user.toString());
+        if (request.getSession() == null) {
+            String sessionId = UUID.randomUUID().toString();
+            sessionManager.add(new Session(sessionId));
+            return create302LoginResponse("/index.html", sessionId);
+        }
```

Comment:
```suggestion
if (request.getSession() == null) {
            String sessionId = UUID.randomUUID().toString();
            Session session = new Session(sessionId);
            session.setAttribute("user", user);
            sessionManager.add(session);
            return create302LoginResponse("/index.html", sessionId);
        }
```

이런식으로 session에 user객체를 저장하는 로직이 추가되어야 할 것 같아요~

지금 sessionManager로 id와 session을 저장하는 이유가 요청을 주고 받을 때 sessionId만으로 로그인 한 user를 기억하기 위해서 인데 session id만 등록을 해놓으면 user를 찾을 수 없을 것 같아요!

Reply:
깜빡했네요! 추가로 구현했습니다~

[feat: 세션에 유저 정보 저장 및 회원 가입 시 세션 생성 구현](https://github.com/woowacourse/java-http/pull/905/commits/a026df8f15546a58f524950a32c3a9170028200d)